deps/build.jl

```diff
@@ -0,0 +1,15 @@
+using Compat
+
+if haskey(ENV, "CONDA_JL_HOME")
+    rootenv = ENV["CONDA_JL_HOME"]
+elseif isfile("deps.jl")
+    include("deps.jl")
+else
+    rootenv = abspath(dirname(@__FILE__), "usr")
+end
+
+deps = "rootenv=\"$(escape_string(rootenv))\""
```

Comment:
`const ROOTENV = ` (should be `const`, and since we are using all-caps for the other path constants we should be consistent here)